MacroTest failing on Windows

When building with `mvn clean install` on Windows (Git Bash), `MacroTest#testBarePluginFromMacro()` fails with:

```
[INFO] Running net.imagej.legacy.MacroTest
Set_Property_Test not up-to-date because 1 source files are not up-to-date (C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\src\main\java\Set_Property_Test.java)
Compiling 1 file in C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252
[-classpath, C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\target\classes;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\access-bridge-64.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\cldrdata.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\dnsns.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\jaccess.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\jfxrt.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\localedata.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\nashorn.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunec.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunjce_provider.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunmscapi.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunpkcs11.jar;C:\Users\eglijan\AppData\Local\Temp\surefire185089902970536562\surefirebooter7785021008127858832.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\zipfs.jar, -d, C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\target\classes, C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\src\main\java\Set_Property_Test.java]
using the class path: C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\target\classes;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\access-bridge-64.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\cldrdata.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\dnsns.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\jaccess.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\jfxrt.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\localedata.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\nashorn.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunec.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunjce_provider.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunmscapi.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\sunpkcs11.jar;C:\Users\eglijan\AppData\Local\Temp\surefire185089902970536562\surefirebooter7785021008127858832.jar;C:\Program%20Files\Java\jdk1.8.0_172\jre\lib\ext\zipfs.jar
C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\src\main\java\Set_Property_Test.java:1: error: package ij does not exist
import ij.IJ;
         ^
C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\src\main\java\Set_Property_Test.java:2: error: package ij.gui does not exist
import ij.gui.GenericDialog;
             ^
C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\src\main\java\Set_Property_Test.java:3: error: package ij.plugin does not exist
import ij.plugin.PlugIn;
                ^
C:\Users\eglijan\AppData\Local\Temp\java7120924943914832252\src\main\java\Set_Property_Test.java:5: error: cannot find symbol
public class Set_Property_Test implements PlugIn {
                                          ^
  symbol: class PlugIn
4 errors
[ERROR] null
java.lang.ClassNotFoundException: Set_Property_Test
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at org.scijava.plugins.scripting.java.JavaEngine.compile(JavaEngine.java:228)
        at org.scijava.plugins.scripting.java.JavaEngine.eval(JavaEngine.java:136)
        at org.scijava.script.ScriptModule.run(ScriptModule.java:160)
        at org.scijava.module.ModuleRunner.run(ModuleRunner.java:168)
        at org.scijava.module.ModuleRunner.call(ModuleRunner.java:127)
        at org.scijava.module.ModuleRunner.call(ModuleRunner.java:66)
        at org.scijava.thread.DefaultThreadService$3.call(DefaultThreadService.java:238)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.253 s <<< FAILURE! - in net.imagej.legacy.MacroTest
[ERROR] testBarePluginFromMacro(net.imagej.legacy.MacroTest)  Time elapsed: 0.155 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[c:\hello\world]> but was:<[E:\code\imagej\imagej-legacy\target\bare-testBarePluginFromMacro-L114]>
        at net.imagej.legacy.MacroTest.testBarePluginFromMacro(MacroTest.java:142)
```

The issue seems to be Windows-specific, as builds on Travis run fine.

--- a/pom.xml
+++ b/pom.xml
@@ -178,6 +178,10 @@
 		<license.projectName>ImageJ2 software for multidimensional image processing and analysis.</license.projectName>
 		<license.excludes>**/script_templates/**,**/doc/**</license.excludes>
 
+		<!-- NB: Required for tests to pass on Windows CLI. -->
+		<!-- see https://github.com/imagej/imagej-legacy/issues/198 -->
+		<maven-surefire-plugin.version>3.1.2</maven-surefire-plugin.version>
+
 		<!-- NB: Deploy releases to the SciJava Maven repository. -->
 		<releaseProfiles>sign,deploy-to-scijava</releaseProfiles>
 	</properties>

